configs/eslint-config-devtools/common.js

```diff
@@ -16,6 +16,7 @@ const tsRules = {
   '@typescript-eslint/no-unsafe-call': 'off',
   '@typescript-eslint/no-unsafe-member-access': 'off',
   '@typescript-eslint/no-unsafe-return': 'off',
+  '@typescript-eslint/no-unsafe-argument': 'warn',
```

Comment:
This became an error by default and there are a LOT of cases of any being passed around.